theme/blockly-toolbox.less

```diff
@@ -7,7 +7,7 @@
 div.blocklyTreeRow {
     margin-bottom: 0px !important;
     transition-property: background-color;
-    transition-duration: 1s;
+    transition-duration: 0.2s;
```

Comment:
Is there a reason this was reduced?

Reply:
@riknoll Mentioned during testing that the on-hover's color transition seemed too long, and ever since he pointed it out, I can't unsee it. It feels really long at 1 second. So I just shortened it.

Reply:
Sounds good!